src/lib/protocol/README.md

```diff
@@ -742,3 +788,18 @@ datagram payload and its byte size.
 
 If *pointer* and *length* are supplied then *length* bytes starting from
 *pointer* are appended to the datagram's payload.
+
+— Method **datagram:data**
+
+Returns the location and size of the buffer of the underlying packet.
+This is a shortcut to *datagram:packet* followed by calls to
+*packet.data* and *pakcet.length*.
```

Comment:
Also note that this introduces a new convenient but strictly speaking redundant public function that we will have to support. Its also questionable whether `datagram:data` will make easier or harder to read, one more thing to have to look up. Is it worth it?

```
data, len = d:data()

  vs

p = d:packet()
data, len = p:data(), p:length()
```